modules/concept-docs/pages/data-durability-acid-transactions.adoc

```diff
@@ -159,6 +159,52 @@ This durability is enforced by the database,
 // https://docs.couchbase.com/java-sdk/current/concept-docs/durability-replication-failure-considerations.html#older-server-versions
 // Change link once 3.3 is archived?
 
+
+
+
+== Reading from Replicas
+
+By default, read requests from the SDK are directed to the node with the active vbucket.
+This bucket will usually be updated before any replicas (see <#durability,Durability> above),
+and is best for consistency.
+
+Some use cases benefit -- for consistency or availability -- from accessing both the active copy of the document, and/or one or all replicas.
+This can be from the {anyreplica-api}[data service],
+and xref:howtos:subdocument-operations.adoc#reading-sub-documents-from-replicas[SubDoc Read from Replica].
+The method can either be one to return either the first result returned -- active or replica --
+which is useful if a node is timing out;
+or to return _all_ of the results, for client code to handle any inconsistency, or to build a consensus answer.
+
+=== Local Availability Zone Reads
```

Comment:
We've been careful to call the feature something like "Preferred Server Group Replica Reads" in case it's used in other contexts in the future. That feature, in combination with careful configuration, can be used for something like "Zone Aware Replica Reads" so I'm wondering if it might be better to introduce it from that perspective instead, and then explain how it can be used for this purpose with all the caveats, etc.